<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviorSettings</class>
 <widget class="QWidget" name="BehaviorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layoutParent">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>615</width>
        <height>486</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="layoutAutofit">
         <item alignment="Qt::AlignLeft">
          <widget class="QCheckBox" name="checkAutofit">
           <property name="toolTip">
            <string>If the player is in windowed mode, the window will be resized to fit the content.</string>
           </property>
           <property name="text">
            <string>Autofit media on load:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QSpinBox" name="spinAutofit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>The percentage to scale video dimensions by.
A value of 0% will fit to media to the window size instead.</string>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="labelAutofitPercent">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerAutofit">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkCursorOSC">
         <property name="toolTip">
          <string>Hides the cursor with the OSC.
If unset, Memento will let mpv control cursor visibility.</string>
         </property>
         <property name="text">
          <string>Hide cursor with OSC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOSC">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>On Screen Controller</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameOSC">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="layoutOSC">
          <item>
           <layout class="QHBoxLayout" name="layoutOSCDuration">
            <item>
             <widget class="QLabel" name="labelOSCDuration">
              <property name="text">
               <string>Visbility duration:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinOSCDuration">
              <property name="toolTip">
               <string>How long the OSC should stay visible without mouse movement.
The fade in duration is included in this time.</string>
              </property>
              <property name="minimum">
               <number>50</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelOSCDurationUnits">
              <property name="text">
               <string>ms</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacerOSCDuration">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutOSCFade">
            <item>
             <widget class="QLabel" name="labelOSCFade">
              <property name="text">
               <string>Fade duration:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinOSCFade">
              <property name="toolTip">
               <string>How long it takes for the OSC to fade in and out</string>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelOSCFadeUnits">
              <property name="text">
               <string>ms</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacerOSCFade">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutOSCMinMove">
            <item>
             <widget class="QLabel" name="labelOSCMinMove">
              <property name="text">
               <string>Minimum mouse movement:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinOSCMinMove">
              <property name="toolTip">
               <string>The number of pixels that need to be moved before the OSC is shown</string>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelOSCMinMoveUnits">
              <property name="text">
               <string>pixels</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacerOSCMinMove">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSubtitles">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Subtitles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSubtitlePause">
         <property name="text">
          <string>Automatically pause at the end of subtitles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSubtitleShow">
         <property name="text">
          <string>Show subtitles when the cursor is at the bottom of the player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSubtitleSecShow">
         <property name="text">
          <string>Show secondary subtitles when the cursor is at the top of the player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOpenFileDir">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The directory the file picker will start at when opening media.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Current&lt;/span&gt; sets the directory to the last directory that was open.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Custom&lt;/span&gt; allows a custom directory to be set.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Open File Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboFileOpenDir">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The directory the file picker will open to.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Current&lt;/span&gt; lets the operating system choose. This is usually the last directory that was opened.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutFileOpenCustom">
         <item>
          <widget class="QLineEdit" name="lineEditFileOpenCustom"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonFileOpenCustom">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
